target/linux/ramips/mt7620/base-files/etc/board.d/01_leds

```diff
@@ -225,6 +225,10 @@ xiaomi,miwifi-mini)
 	ucidef_set_led_switch "lan2" "lan2" "green:lan2" "switch0" "0x01"
 	ucidef_set_led_switch "wan" "wan" "green:wan" "switch0" "0x10"
 	;;
+bolt,bl201)
+	ucidef_set_led_netdev "wlan0" "wlan0-link" "blue:wlan" "wlan0"
```

Comment:
Why do you use the name "wlan0-link" here? I think this will also trigger on rx and tx with this configuration.